packages/@lwc/engine-core/src/framework/invoker.ts

```diff
@@ -26,6 +26,9 @@ export function isInvokingRenderedCallback(vm: VM): boolean {
     return vmInvokingRenderedCallback === vm;
 }
 
+let profilerEnabled = false;
+trackProfilerState((t) => (profilerEnabled = t));
```

Comment:
Pasting this from comment on [older PR](https://github.com/salesforce/lwc/pull/1983/files#r458294187)
![image](https://user-images.githubusercontent.com/21322483/88104822-dbe48000-cb57-11ea-91ac-ee6582fe7ffc.png)